responsive-grid-layout/savas/global.css

```diff
@@ -0,0 +1,144 @@
+* {
+    box-sizing: border-box;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+}
+
+body {
+    background-color: #f1f2f3;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    margin: 0;
+}
+
+.container {
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+}
+
+.header {
+    background-color: #006c67;
+    color: white;
+}
+
+.sidebar-left {
+    background-color: #003844;
+    color: white;
+}
+
+.content {
+    background-color: #ffebc6;
+}
+
+.sidebar-right {
+    background-color: #ffb100;
+}
+
+.footer {
+    background-color: #006c67;
+    color: white;
+}
+
+@media all and (max-width: 768px) {
```

Comment:
For device size specific code (defined in media query selectors), you can use the "responsive.css" file. This separation would make the code for bigger projects more readable. 

Then, just make sure that you css files are added to the HTML file in the correct order: from more generic to more specific:
```
<link rel="stylesheet" href="./global.css">
<link rel="stylesheet" href="./responsive.css">

```

Reply:
I got it and moved these codes to responsive.css file.